### PR TITLE
Add Bolivian PagosNet

### PR DESCRIPTION
**Pull request by afstarosta**

### CODE REVIEW

File: tests/woocommerce/lib/shop/pages/checkout.js [hunk 1]
```diff
@@ -367,7 +367,6 @@ export default class Checkout {
       R.propSatisfies((x) => (x !== undefined), pM.property), (data) => {
         this.cy
           .get(pM.elm(data[pM.property], data.countryId.toLowerCase()))
-          .should('be.visible')
```
Comment (afstarosta) [inline tests/woocommerce/lib/shop/pages/checkout.js]: Payment Selection button is only visible for countries with more than one Payment Method. Since Bolivia is the first to have only one, this line makes the test fail.

File: gateways/class-wc-ebanx-pagosnet-gateway.php [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+	 * Save order's meta fields for future use
+	 *
+	 * @param  WC_Order $order The order created.
+	 * @param  Object   $request The request from EBANX success response.
+	 * @return void
+	 */
+	protected function save_order_meta_fields( $order, $request ) {
+		parent::save_order_meta_fields( $order, $request );
+
+		update_post_meta( $order->id, '_voucher_url', $request->payment->voucher_url );
```
Comment (Ruanito) [inline gateways/class-wc-ebanx-pagosnet-gateway.php]: You should use the method `get_id()`

File: templates/pagosnet/payment-failed.php [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+ * @author  EBANX.com
+ * @package WooCommerce_EBANX/Templates
+ * @version 2.0.0
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+
```
Comment (Ruanito) [inline templates/pagosnet/payment-failed.php]: remove this line

File: templates/pagosnet/payment-processing.php [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+ * @author  EBANX.com
+ * @package WooCommerce_EBANX/Templates
+ * @version 2.0.0
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+
```
Comment (Ruanito) [inline templates/pagosnet/payment-processing.php]: remove this line

File: woocommerce-gateway-ebanx.php [hunk 5]
```diff
@@ -664,6 +672,9 @@ private function includes() {
 			include_once WC_EBANX_GATEWAYS_DIR . 'class-wc-ebanx-oxxo-gateway.php';
 			include_once WC_EBANX_GATEWAYS_DIR . 'class-wc-ebanx-spei-gateway.php';
 
+			// Bolivia Gateways.
+			include_once WC_EBANX_GATEWAYS_DIR . 'class-wc-ebanx-pagosnet-gateway.php';
```
Comment (Ruanito) [inline woocommerce-gateway-ebanx.php]: use namespace instead of using `include_once`.

File: gateways/class-wc-ebanx-pagosnet-gateway.php [hunk 6]
```diff
@@ -0,0 +1,151 @@
...
+	 * @return void
+	 */
+	public static function thankyou_page( $order ) {
+		$voucher_url     = get_post_meta( $order->get_id(), '_voucher_url', true );
+		$voucher_basic   = $voucher_url . '&format=basic';
+		$voucher_pdf     = $voucher_url . '&format=pdf';
+		$voucher_print   = $voucher_url . '&format=print';
+		$customer_email  = get_post_meta( $order->get_id(), '_ebanx_payment_customer_email', true );
+		$voucher_hash    = get_post_meta( $order->get_id(), '_ebanx_payment_hash', true );
+		$customer_name   = $order->billing_first_name;
```
Comment (Ruanito) [inline gateways/class-wc-ebanx-pagosnet-gateway.php]: You should use the function to get the `billing_first_name`

File: lib/Plugin/Services/WC_EBANX_Payment_Adapter.php [hunk 7]
```diff
@@ -8,6 +8,7 @@
 use Ebanx\Benjamin\Models\Item;
 use Ebanx\Benjamin\Models\Payment;
 use Ebanx\Benjamin\Models\Person;
+use Exception;
```
Comment (Ruanito) [inline lib/Plugin/Services/WC_EBANX_Payment_Adapter.php]: Why did you add this `Exception`
